Makefile

```diff
@@ -15,4 +15,8 @@ fmt:
 
 .PHONY: build
 build:
-	go build -o bin/cq-source-sharepoint -v
\ No newline at end of file
+	go build -o bin/cq-source-sharepoint -v
+
+.PHONY: package
+package:
+	go run main.go package --docs-dir docs -m @CHANGELOG.md v2.0.0 .
```

Comment:
I guess it should better be a part of a pipeline (GitHub Action: .github/workflows/release.yaml, as a post release action) which used the git tag as a version instead a hardcoded one.

Reply:
Yeah, committed it to the Makefile to.make it easier to see & not forget about